docs/learning-powershell/README.md

```diff
@@ -10,6 +10,13 @@ You won't be a PowerShell guru at the end of reading this material but you will
 
 If you have 30 minutes now, let’s try it.
 
+Advantages of Using PowerShell
```

Comment:
@razinc 	or @RDIL 	

Could you clean up this file with markdownlint?

The rules are in the root of the repo.  There is a Visual Studio code extension for markdown lint which works well if you open from the root of the repo.